src/content/docs/index.md

```diff
@@ -165,7 +165,7 @@ console.log(capability.inc()); // 2
 ## Boundaries
 
 Hardened JavaScript does not protect the availability of a program.
-Any party in the same realm, regardless of compartment isolation, can drop into
+Any party in the same agent, regardless of compartment isolation, can drop into
```

Comment:
again this assumes that there exists multiple realms.

Reply:
Since this document’s audience include folks who aren’t intimate with the term “agent”, can I suggest “process”? I think “agent” and even “worker” suffer a distinction, where the spectre/meltdown mitigation needs a _bona fide_ process boundary and a host could even have multiple “agent clusters” in a single process.